app/index.jade

```diff
@@ -42,8 +42,8 @@ body(ng-controller="AppCtrl" ui-view="body" ng-mousemove="resetInactivityTime()"
   //- This does not depend on Angular and will show as soon as the images are done loading.
   .blocket-loading
     div
-      img.full(src="img/blocket-full.png")
-      img(src="img/blocket-no-boosters.png")
+      img.full(src="img/blocket-full.png" alt="Loading")
```

Comment:
Since we're loading this before Angular doesn't that stop us from being able to us translate here?